cmd/av/stack_branchcommit.go

```diff
@@ -43,8 +46,9 @@ var stackBranchCommitFlags struct {
 
 var stackBranchCommitCmd = &cobra.Command{
 	Use:          "branch-commit [flags]",
-	Short:        "create a new stacked branch and a commit",
-	Long:         "Create a new branch that is stacked on the current branch, and call git-commit with the specified arguments.",
+	Aliases:      []string{"bc"},
+	Short:        "switch to a new stacked branch commit staged changes to it",
```

Comment:
This is very confusing right now.
```suggestion
	Short:        "create a new stacked branch and commit staged changes to it",
```